src/goddard/renderer.c

```diff
@@ -2328,7 +2337,7 @@ void gd_set_one_cycle(void) {
 }
 
 /* 250B30 -> 250B44 */
-void stub_renderer_3(void) {
+UNUSED void stub_renderer_3(void) {
```

Comment:
Keeping the unused functions is up for debate already but this one definitely serves no purpose and should just be deleted.